Gordon360/Controllers/LostAndFoundController.cs

```diff
@@ -36,6 +36,21 @@ public async Task<ActionResult> UpdateMissingItemReport(int id, [FromBody] Missi
             return Ok();
         }
 
+        /// <summary>
+        /// Update Item Report Status to "Deleted" with the given id
+        /// </summary>
+        /// <param name="id">The id</param>
+        /// <returns></returns>
+        [HttpPut]
+        [Route("missingitem/status")]
+        public async Task<ActionResult> UpdateReportStatus(int id, [FromBody] MissingItemReportViewModel MissingItemDetails)
```

Comment:
This probably shouldn't take the whole Missing item details object, it should just take an id and a status probably